site/LAYOUT.md

```diff
@@ -0,0 +1,22 @@
+# Site layout
+
+The update center image contains the following pieces.
+
+ * Version specific update sites like `1.xxx` and `2.xxx`: update sites that advertises the versions of plugins that are supported for different version ranges of masters. See [README.md](README.md).
+ * `current`: update site for the rest of masters, which advertises the latest of everything.
+ * [`/download` tree](http://updates.jenkins-ci.org/download): an URL space that covers all the versions of all the plugins released to date. Thoes URLs are then redirected to `mirrors.jenkins-ci.org`
```

Comment:
Mind changing this to `mirrors.jenkins.io., I would like to remove all jenkins-ci.org references from our minds :P